.github/workflows/build-push-images-azure.yml

```diff
@@ -44,11 +44,9 @@ jobs:
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
       - run: |
-          docker build ./src --file ./src/prod.Dockerfile --tag ${{ secrets.REGISTRY_LOGIN_SERVER }}/wagtail:${{ github.ref_name }} --target wagtail
-          docker build ./src --file ./src/prod.Dockerfile --tag ${{ secrets.REGISTRY_LOGIN_SERVER }}/clean-scenarios:${{ github.ref_name }} --target clean-scenarios
-          docker build ./frontend --build-arg NEXT_PUBLIC_WAGTAIL_API_URL=$NEXT_PUBLIC_WAGTAIL_API_URL -t ${{ secrets.REGISTRY_LOGIN_SERVER }}/nextjs:${{ github.ref_name }} -f ./frontend/prod.Dockerfile
```

Comment:
Thanks @Erikvv! I squashed all commits regarding the table logic together and ditched the merge/reverted merge, but now something with the other two merge commits went a bit strange... Hope that's fine for you!

Have to say that I have not the most experience with merge commits, at the ETM we're big rebasers 😉 Otherwise, I can also squash the ones from pascal, ditch the two from you and just rebase on main

Reply:
Thank you.